config/jobs/kubernetes-sigs/kind/kind.yaml

```diff
@@ -40,7 +40,10 @@ periodics:
     repo: kubernetes
     base_ref: master
     path_alias: k8s.io/kubernetes
-  path_alias: sigs.k8s.io/kind
+  - org: kubernetes-sigs
+    repo: kind
+    base_ref: master
+    path_alias: sigs.k8s.io/kind
```

Comment:
The CI jobs should be using the latest binaries and e2e-k8s.sh like kubernetes presubmits. More clones and building is slower, flakier, etc.

Reply:
ok, let me send a new PR

Reply:
https://github.com/kubernetes/test-infra/pull/17596